Automattic/jetpack
# Required Review Action: retrieve GitHub server URL from context

This is required in order to correctly support GitHub Enterprise usage, otherwise the check detail links are broken.

Fixes #32973

## Proposed changes:
* Retrieve GitHub server URL from the context instead of hardcoding it in as github.com

## Testing instructions:

* Utilize the required-review GitHub Action in any Actions workflow with a sample configuration
* Create a pull request
* Click "Details" next to the Require Review check in the CI section

Example:
```
---
name: Required Review test
on: pull_request

jobs:
  check:
    name: Required Reviews
    runs-on: ubuntu
    steps:
      - uses: Automattic/action-required-review@v3
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          requirements: |
            - name: All files
              paths:
                - '**'
              teams:
                - test-team
```

## Does this pull request change what data or activity we track or use?
No

## Affected projects
GitHub Action for required review
